docs/source/contribution/technical_steering_committee.md

```diff
@@ -56,28 +56,23 @@ We look for commitment markers who can do the following:
 | Name                                                     | Organisation                                                                            |
```

Comment:
Do we need TSC votes to _remove_ members from the TSC?

cc @yetudada

Reply:
We haven't in the past. The charter is a PDF in the root of the repo but I scanned it and it wasn't clear on leaving (only that 2/3 should approve changes to the charter). For QB members, I would say that if someone leaves the team they are voluntarily leaving as a TSC member, unless they explicitly commit to continue contributing and do so by regularly through meeting attendance and/or code contributions/PR reviews. 

It's worth asking each person if they intend to do that.